### DISCUSSION TITLE
Cannot access API from Apache after removing hash tag

### DISCUSSION THREAD

**Discussion by toprater** [community/community#53974] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Select Feature Area

- [ ] Issues
- [ ] Projects

### Body

When deploying an Angular v15 application to an Apache-based hosting, I had some issues with removal of # hashtag from the URL, because once it is switched off in the client, the server fails to load correct pages on refresh, and produces a 404 error.
I guess this was because of `you must configure the server to return the application's host page (index.html) when asked for a file that it does not have`

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Juiux** [comment, marked as answer]
A fix to skip the /api/ URLs. You can add this:

RewriteCond %{REQUEST_URI} !^/api/

Setting a condition for the rule to not apply if the requested URI starts with /api/.
